src/index.ts

```diff
@@ -56,7 +56,6 @@ declare const formatDate;
 
 export * from "./types/Icon.types";
 export {
-  // typed
```

Comment:
The distinction between typed and untyped components in this `export` doesn't really matter as long as an export is present for each component.

Simplifying this